components/Table/Table.js

```diff
@@ -85,7 +96,63 @@ export class Table {
       menuBtn.classList.add('menuBtn');
       menuBtn.textContent = 'more_vert';
 
-      actionsContainer.append(arrowBtn, menuBtn);
+      const dropDownMenu = document.createElement('div');
+      dropDownMenu.classList.add('drop-down__menu');
+      const menuItems = this.menuConfig.map((item) => {
```

Comment:
Since we have a long code here, please add a more descriptive name instead of `item`, like `menuItem` or so